src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingSourceGenerator.cs

```diff
@@ -42,29 +49,30 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
         /// </summary>
         private static void Execute(CompilationData compilationData, ImmutableArray<BinderInvocationOperation> inputCalls, SourceProductionContext context)
         {
-#if LAUNCH_DEBUGGER
-            if (!System.Diagnostics.Debugger.IsAttached)
-            {
-                System.Diagnostics.Debugger.Launch();
-            }
-#endif
             if (inputCalls.IsDefaultOrEmpty)
             {
                 return;
             }
 
             if (compilationData?.LanguageVersionIsSupported != true)
             {
-                context.ReportDiagnostic(Diagnostic.Create(LanguageVersionNotSupported, location: null));
+                context.ReportDiagnostic(Diagnostic.Create(Helpers.LanguageVersionNotSupported, location: null));
                 return;
             }
 
-            Parser parser = new(context, compilationData.TypeSymbols!);
-            SourceGenerationSpec? spec = parser.GetSourceGenerationSpec(inputCalls);
-            if (spec is not null)
+            try
+            {
+                Parser parser = new(context, compilationData.TypeSymbols!);
+                SourceGenerationSpec? spec = parser.GetSourceGenerationSpec(inputCalls);
+                if (spec is not null)
+                {
+                    Emitter emitter = new(context, spec);
+                    emitter.Emit();
+                }
+            }
+            catch (System.Exception ex)
             {
-                Emitter emitter = new(context, spec);
-                emitter.Emit();
+                ex.ToString();
```

Comment:
>  ex.ToString(); [](http://example.com/codeflow?start=14&length=16)

How this is helping in anything?

Reply:
Was for debugging locally. Will remove/clean up.